Source/WebCore/page/PerformanceObserver.h

```diff
@@ -83,10 +83,10 @@ class PerformanceObserver : public RefCounted<PerformanceObserver> {
     Vector<Ref<PerformanceEntry>> m_entriesToDeliver;
     const Ref<PerformanceObserverCallback> m_callback;
     OptionSet<PerformanceEntry::Type> m_typeFilter;
+    Seconds m_durationThreshold { 0 };
```

Comment:
No need to initialize this explicitly to 0. The default constructor does that automatically.